src/org/usfirst/frc/team4571/robot/commands/MillCommand.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc.team4571.robot.commands;
+
+import org.usfirst.frc.team4571.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ *
+ */
+public class MillCommand extends Command {
+
+    public MillCommand() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    	requires(Robot.MILL_SUBSYSTEM);
+    }
+    public void forward(){
+    	Robot.MILL_SUBSYSTEM.forward(SmartDashboard.getDouble("MillForwardSpeed", 0.5));
+    }
+    public void reverse(){
+    	Robot.MILL_SUBSYSTEM.reverse(SmartDashboard.getDouble("MillReverseSpeed", -0.5));
```

Comment:
Do we write the SmartDashboard line under both the forward and reverse methods?
And do we make the value in the key-value pair for the reverse method negative for SmartDashboard, or do we leave that value positive?